sdk/search/azure-search-documents/azure/search/documents/_index/aio/_search_index_client_async.py

```diff
@@ -19,7 +19,7 @@
 if TYPE_CHECKING:
     # pylint:disable=unused-import,ungrouped-imports
     from typing import Any, Union
-    from .._credential import SearchApiKeyCredential
+    from ... import SearchApiKeyCredential
```

Comment:
from <ellipsis> import <something> ? Is this GitHub being odd with what has changed? Or is this actually a python quirk that makes `...` the same as `..` in this context?

Reply:
Huh. 

```shell$ python
Python 3.8.0 (default, Oct 18 2019, 18:23:33) 
[Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from  ... import *
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: attempted relative import with no known parent package
>>> 
```

Reply:
Not the error message I expected :)

Reply:
@johanste I don't understand the question I decided to import from the public API `azure.search.document` which is three levels up (i.e. `...`) instead of from the private module `..._credential`

Reply:
note that I moved `_crediential.py` one more level up (so it can be used by both `_index` and by `_service` in case that is what is confusing

Reply:
Sorry - I guess I had never imported something relative two levels up :). I assumed the syntax would be `from ../.. import`. 

Today I learned something new.